src/views/Home.vue

```diff
@@ -1,18 +1,142 @@
 <template>
   <div class="home">
-    <img alt="Vue logo" src="../assets/logo.png">
-    <HelloWorld msg="Welcome to Your Vue.js + TypeScript App"/>
+    <div class="input">
+      <InputField
+        type="text"
+        placeholder="First Name"
+        name="firstName"
+        :invalid-input="submitted && v$.user.firstName.$invalid"
+        :error-message="`${$t('validationError.firstName')}`"
+        @changeValue="updateUser"
+      />
+    </div>
+    <div class="input">
+      <InputField
+        type="text"
+        placeholder="Last Name"
+        name="lastName"
+        :invalid-input="submitted && v$.user.lastName.$invalid"
+        :error-message="`${$t('validationError.lastName')}`"
+        @changeValue="updateUser"
+      />
+    </div>
+    <div class="input">
+      <InputField
+        type="email"
+        placeholder="Email"
+        name="email"
+        :invalid-input="submitted && v$.user.email.$invalid"
+        :error-message="`${$t('validationError.email')}`"
+        @changeValue="updateUser"
+      />
+    </div>
+    <div class="input">
+      <InputField
+        type="password"
+        placeholder="Password"
+        name="password"
+        :invalid-input="submitted && v$.user.password.$invalid"
+        :error-message="`${$t('validationError.password')}`"
+        @changeValue="updateUser"
+      />
+    </div>
+    <div class="input">
+      <InputField
+        type="password"
+        placeholder="Confirm Password"
+        name="confirmPassword"
+        :invalid-input="submitted && v$.user.confirmPassword.$invalid"
+        :error-message="`${$t('validationError.confirmPassword')}`"
+        @changeValue="updateUser"
+      />
+    </div>
+    <Button :message="`${$t('button.register')}`" @onClick="registerUser" />
```

Comment:
Passing message like this feels unintuitive as its not a property.

Is it possible to achieve, something to this effect?
`<Button  @onClick="registerUser" />Message</Button>`